dashboard/test/controllers/pd/professional_learning_landing_controller_test.rb

```diff
@@ -13,7 +13,11 @@ class Pd::ProfessionalLearningLandingControllerTest < ::ActionController::TestCa
       create :pd_enrollment, email: other_teacher.email, workshop: workshop
     end
 
-    create :pd_enrollment, email: @teacher.email, workshop: @csf_workshop
+    @ended_enrollment = create :pd_enrollment, email: @teacher.email, workshop: @csf_workshop
+    create :pd_enrollment, email: @teacher.email, workshop: @csd_workshop
+
+    @csf_workshop.start!
+    @csf_workshop.end!
```

Comment:
Note this affects other enrollments in the csf workshop. It _might_ be better to leave those 3 workshops alone and create an addition workshop (or workshops) that are already ended. Your call.

Also note, you can create a workshop that is already ended with [create :pd_ended_workshop](https://github.com/code-dot-org/code-dot-org/blob/c67c4061e5ba0d13f087fe45e56ffed889e877f1/dashboard/test/factories.rb#L537-L542)